api/src/pcapi/core/offerers/update_offerer_stats.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+import logging
+
+from pcapi.connectors.big_query.queries.offerer_stats import DAILY_CONSULT_PER_OFFERER_LAST_180_DAYS_TABLE
+from pcapi.connectors.big_query.queries.offerer_stats import OffererDailyViewsLast180Days
+from pcapi.connectors.big_query.queries.offerer_stats import OffererTopOffersAndTotalViewsLast30Days
+from pcapi.connectors.big_query.queries.offerer_stats import PAGE_SIZE
+from pcapi.connectors.big_query.queries.offerer_stats import TOP_3_MOST_CONSULTED_OFFERS_LAST_30_DAYS_TABLE
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+from pcapi.repository import transaction
+
+
+logger = logging.getLogger(__name__)
+
+
+def update_offerer_daily_views_stats() -> None:
+    fetched_all_data = False
+    offset = 0
+    emptied_table = False
+    while not fetched_all_data:
```

Comment:
Utiliser une boucle for sur les resultats de la requête OffererDailyViewsLast180Days. 
Reproduire le schéma de https://github.com/pass-culture/pass-culture-main/blob/621b17df1bf4fa7c96e181e77eed10cb9053c3de/api/src/pcapi/scheduled_tasks/commands.py#L433